src/Riak/Command/Builder/TimeSeries/DescribeTable.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Basho\Riak\Command\Builder\TimeSeries;
+
+use Basho\Riak;
+use Basho\Riak\Command;
+
+/**
+ * @author Christopher Mancini <cmancini at basho d0t com>
+ */
+class DescribeTable extends Command\Builder\TimeSeries\Query implements Command\BuilderInterface
+{
+    /**
+     * Which table do you want to describe?
+     *
+     * @param $table
+     */
+    public function withTable($table)
```

Comment:
Mainly because the context is different than the tabletTrait, e.g. inTable doesn't quite make sense for describing a tables schema. Also, it extends the query builder and stores it as a query in the object, not a table value.